src/constants/networks.ts

```diff
@@ -0,0 +1,23 @@
+const NETWORKS = {
```

Comment:
The list of available networks should be fetched from the point node instead of being hard coded here.